main.go

```diff
@@ -112,7 +112,7 @@ func (c collectorSet) isEmpty() bool {
 }
 
 func (c *collectorSet) Type() string {
-	return "map[string]struct{}"
```

Comment:
We should not change the type string here. 

With using `fmt.Sprintf("%v", &defaultCollectors)`, we can also get the default collectors `"services,persistentvolumeclaims,nodes,replicationcontrollers,statefulsets,replicasets,resourcequotas,jobs,namespaces,daemonsets,deployments,limitranges,pods,cronjobs"`.

Reply:
I know, but I feel it is more understandable for me that Type method returns `string` than `map[string]struct{}`.

"string"
```
      --collectors string                Comma-separated list of collectors to be enabled. Defaults to "services,persistentvolumeclaims,nodes,replicationcontrollers,statefulsets,replicasets,resourcequotas,jobs,namespaces,daemonsets,deployments,limitranges,pods,cronjobs"
```

"map[string]struct{}"
```
      --collectors map[string]struct{}   Comma-separated list of collectors to be enabled. Defaults to "nodes,replicationcontrollers,cronjobs,persistentvolumeclaims,namespaces,daemonsets,jobs,limitranges,replicasets,resourcequotas,services,deployments,statefulsets,pods"
```

What do you think?

Reply:
I think for non go developers it's hard to understand that `map[string]struct{}` is a set, so I think the method @andyxning suggested will be easier to understand for users.

Reply:
> I think for non go developers it's hard to understand that map[string]struct{} is a set

I think what @brancz means is that what @superbrothers suggests is more friendly to end users. :) So, 

/lgtm

Reply:
Sorry yes, that was correct I mixed up who said what :slightly_smiling_face:, thanks for the contribution @superbrothers !